main.dart

```diff
@@ -0,0 +1,17 @@
+import 'Utills/Utillty.dart';
+main(List<String> args) {
+
+  Map<int,Map<String,dynamic>> data = UtilityIO.getDatabaseData();
```

Comment:
should specify what data or what type of data. 

Also, Not required to mention database in function, that can be abstract. So that even if implementation changes in future, you need not change the function name.
Just getUserData or something works fine.